advanced_html_css/accessibility/accessibility_auditing.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-By now you should feel confident enough to start making your websites a little more accessible for a lot of users. In this lesson, you will learn to check the implementation of certain a11y features with the help of DevTools.
+By now you should feel confident enough to start making your websites a little more accessible for a lot of users. In this lesson, you will learn how to if you've correctly implemented your a11y features with the help of DevTools.
```

Comment:
I removed the `by viewing accessibility tree.. audit pages... ` part of the original sentence (see the very first commit of this PR) because those are mentioned in `Learning outcomes` already. I thought it is enough for the Introduction to introduce what we are going to do in the lesson in a bigger picture rather than getting to the specific 'how's here. Feel free to edit it if you feel otherwise.

Reply:
The prior wording makes far more sense in the terms of this lesson, as the discussed tools, axe and lighthouse are used to "check" implementation of a11y criterions and here, they are being discussed about- thus, we're discussing about tools that help to check for proper a11y guidelines.
@Eveieve

Reply:
@Lofty-Brambles Are you talking about these two particular commits? Or the very first one and the one I made very last?

Reply:
The one change in line 2

Reply:
I feel like we could probably redo this entire intro... Introductions can be difficult, but maybe instead of conveying "This lesson you'll learn XYZ" only to go into the next section which also says that, we could change how we convey the info. Maybe something like:

"By now you should feel confident enough to start making your websites a little more accessible for a lot of users. "How do I know if there are any more issues?" I hear you ask. "How can I inspect accessibility related properties of a specific element?" is another question I think I heard from the back. In this lesson we'll be answering both of these questions to help push your a11y skills over the top."

(Off the top of my head + trying to have some fun with it 😆 But basically we could probably tweak this intro a bit more than what it currently is)

Reply:
@Eveieve in case you hadn't seen this, wdyt? I think what you have right now works, but if you wanted an alternative

Reply:
Hi @thatblindgeye, I think redoing the Introduction for this lesson makes sense because the Introduction works like a 'hook' in other lessons instead of merely repeating what we have in the Lesson overview.  I like what you have! 

I took some inspiration from your draft and come up with my version. 

`Now that you are equipped with the necessary knowledge to make your websites more accessible to many users, the question arises: How can we verify the correct implementation of a11y features? Are there any mistakes to be corrected, or potential improvements to be made?  In this lesson, we will address those questions to help push your a11y skills over the top.`

I wanted it to be concise and to the point yet inviting enough to grab the learner's attention. "By now you should be confident enough... " is a sentence I see often in other TOP lessons so I wanted to give it some variety. What do you think? 

If there's any suggestion you'd like to make, feel free to do so, both with your draft or mine. When I made changes to the Introduction, I mainly wanted to change it so that the paragraph itself is not confusing and not repeating itself. I think it's important to keep the writing non-complicated and concise. So if that's achieved, I'm flexible with other things.

Reply:
Sounds good to me!

Reply:
Great, I'll commit this now!

Reply:
@thatblindgeye I edited the file and have `commit` the change. It's save to `Resolve conversation` for this review, right?

Reply:
Yep you could! No real hard rule on that